test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -72,7 +72,7 @@ function mixing_length(
         m.turbconv.surface.zLL,
     )
 
-    L_W = ml.κ * z / (sqrt(m.turbconv.surface.κ_star²) * ml.c_m)
+    L_W = ml.κ * max(z, 5) / (sqrt(m.turbconv.surface.κ_star²) * ml.c_m)
```

Comment:
This change is needed to avoid devision by a small number near the surface. for some reason it appears in FV button in DG